hack/profile.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+
+# use time to record the elapsed time for the step
+shift # get rid of the '-c' supplied by make.
+
+if [[ -z ${KUBEVIRT_PROFILE_MAKE} ]]; then
+    /bin/bash -c "$*"
+else
+    time /bin/bash -c "$*"
```

Comment:
Would it make sens to just use `date`? I think that would make it easier to detect where we are stalling.

Reply:
Done.